src/redis/types.rs

```diff
@@ -59,13 +59,13 @@ pub struct FeatureDimension {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
 pub enum DimensionType {
```

Comment:
```suggestion
#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
pub enum DimensionType {
```
rename all other variants